libs/surfaces/console1/console1.cc

```diff
@@ -197,6 +204,22 @@ Console1::begin_using_device ()
 	return MIDISurface::begin_using_device ();
 }
 
+int
+Console1::stop_using_device ()
+{
+	DEBUG_TRACE (DEBUG::Console1, "stop_using_device()\n");
+	if (!_in_use)
+		return 0;
+	blink_connection.disconnect ();
+	periodic_connection.disconnect ();
+	stripable_connections.drop_connections ();
+	session_connections.drop_connections ();
+	console1_connections.drop_connections ();
+	MIDISurface::stop_using_device ();
+	_in_use = false;
+	return 0;
+}
+
```

Comment:
Why add this back? instead of using `MIDISurface::stop_using_device`?

Reply:
I moved this in my first PR because I found it much clearer to have the two methods start_using_device and stop_using_device directly under each other. 
This makes it easier to keep track of what has to be done when the status of the device changes.
Keep in mind that I don't have the experience and the overview that you do. (Which is pretty obvious right now;-)

Reply:
_in_use = false removed